docs/source/users/options.rst

```diff
@@ -4,179 +4,28 @@
 FitBenchmarking Options
 #######################
 
-FitBenchmarking is controlled by 2 sets of options,
-which are controlled in 2 ways, software arguments and software options.
+FitBenchmarking is controlled by a set of options that can be controlled in
+3 different ways.
+In order from lowest priority to highest these are:
 
-Software Arguments
-==================
-The first set of options are set in the example scripts and used as arguments
-for the software.
-These include:
+- The default options.
+- An options file.
+- Options set in a script.
 
-``software``
-------------
-The software to use in fitting as a string or list of strings.
-Selected softwares will be benchmarked.
+The default options are a complete set of options with sensible values.
+These will be used when no other values are given for any of the options,
+the values for these can be seen at the end of this document.
 
-Available options are ``"dfogn"``, ``"mantid"``, ``"minuit"``, ``""ralfit""``, ``"sasview"``,
-and ``"scipy"``.
+The options file must be a `.ini` file, and a good reference for this can be
+found in the examples, as well as at the bottom of this document.
```

Comment:
Maybe worth adding a link to a formatting page?

Reply:
done